saleor/plugins/webhook/tests/test_webhook.py

```diff
@@ -148,6 +149,25 @@ def test_product_updated(mocked_webhook_trigger, settings, product):
     )
 
 
+@mock.patch("saleor.plugins.webhook.plugin.trigger_webhooks_for_event.delay")
+def test_product_deleted(mocked_webhook_trigger, settings, product):
+    settings.PLUGINS = ["saleor.plugins.webhook.plugin.WebhookPlugin"]
+    manager = get_plugins_manager()
+
+    product = product
+    variants_id = product.variants.all().values_list("id", flat=True)
+    product_id = product.id
+    product.delete()
+    product.id = product_id
+
+    manager.product_deleted(product, product.variants.all())
+
+    expected_data = generate_product_deleted_payload(product, variants_id)
```

Comment:
I put ipdb here and check how `expected_data` look. `Variants` in `expected_data` are empty :(
![image](https://user-images.githubusercontent.com/43955230/106260532-ca73e300-6220-11eb-97cb-7380432423c6.png)

Reply:
In `expected_data` are missing fields compare to `PRODUCT_FIELDS`

Reply:
Its should be good now. You can see payload from my mock server https://github.com/mirumee/saleor/pull/6794#discussion_r566707941

Reply:
Still same result (`variants` is empty).
I think we should add 2 asserts. 
1. product id is not empty
2. variants is not empty

Reply:
3. `variants` is a list of GraphQL `ID`s and not a list of integers.